### DISCUSSION TITLE
Are the v2.x docs still available online?

### DISCUSSION THREAD

**Discussion by ajrussellaudio** [TanStack/query#1434] [General] [answered]
v3 looks great, and we're exciting about upgrading. But we're not at a stage where it's convenient to migrate yet.

Are the v2 docs still available online, or do we need to build them ourselves from the git history?

**Comment by TkDodo** [comment, marked as answer]
see: https://github.com/tannerlinsley/react-query/issues/1420

**Reply by ajrussellaudio** [reply]
Thank you!
